src/clanhr/ring_test_client/core.clj

```diff
@@ -5,6 +5,12 @@
   (:use clojure.test
         ring.mock.request))
 
+(defn ->edn [raw]
```

Comment:
@lamuria it's idiomatic when you don't know the input format (and that's the case). This converts the input(end, string, stream) to edn.